src/plugins/UtilCommands/controllers/commands/getWarnings.js

```diff
@@ -0,0 +1,20 @@
+const { User, Warning } = require("./../../../../../api/models/index.js");
```

Comment:
Usually a long relative path(`./../../../../..`) is not recommended.

Reply:
Do you recommend using an absolute path?

Reply:
I'm not familiar with the JS import system. Can you use absolute path?